lib/Graph/Graph.cpp

```diff
@@ -941,6 +941,35 @@ IntLookupTableNode *Function::createIntTanh(llvm::StringRef name,
   return createIntLookupTable(name, input, mapping, outTy);
 }
 
+IntLookupTableNode *Function::createIntSigmoid(llvm::StringRef name,
+                                               NodeValue input, TypeRef outTy) {
+  static int8_t mapping[] = {
```

Comment:
This table maps numbers from [-128, 127] to [-116, 115]. Why it is not using the whole int8 range?

Reply:
Values 0 and 1 should not be represented, so -128 and 127 probably should not be in there.
I generated this table based on the fp32 values and then converted them back to int8 with {s,o} which correspond to [0,1] range.

Reply:
So if Sigmoid sees -1000.0 float, it will be rescaled to -128 quantized, and then translated to -116, which corresponds to something much larger then 0 (presumably 0.0474259). Isn't it better to translate to -128 (0.0 float) ?

I think that if we extend range from [-3, +3], we'll have [-128, 127] in the table.

Reply:
Regenerated this for [-6.0, 6.0] interval now.